src/Avalonia.Themes.Fluent/FluentTheme.cs

```diff
@@ -105,10 +142,47 @@ public bool TryGetResource(object key, out object? value)
         void IResourceProvider.AddOwner(IResourceHost owner) => (Loaded as IResourceProvider)?.AddOwner(owner);
         void IResourceProvider.RemoveOwner(IResourceHost owner) => (Loaded as IResourceProvider)?.RemoveOwner(owner);
 
-        private Uri GetUri() => Mode switch
+        private void InitStyles(Uri baseUri)
         {
-            FluentThemeMode.Dark => new Uri("avares://Avalonia.Themes.Fluent/FluentDark.xaml", UriKind.Absolute),
-            _ => new Uri("avares://Avalonia.Themes.Fluent/FluentLight.xaml", UriKind.Absolute),
-        };
+            _sharedStyles = new Styles
+            {
+                new StyleInclude(baseUri)
+                {
+                    Source = new Uri("avares://Avalonia.Themes.Fluent/Accents/AccentColors.xaml")
+                },
+                new StyleInclude(baseUri)
+                {
+                    Source = new Uri("avares://Avalonia.Themes.Fluent/Accents/Base.xaml")
+                },
+                new StyleInclude(baseUri)
+                {
+                    Source = new Uri("avares://Avalonia.Themes.Fluent/Controls/FluentControls.xaml")
```

Comment:
In this way control styles are initialized before control resources, opposite of what was before. 
It might be okay, but have you checked if there any static resources that might be missed in this approach?

Reply:
May you check also? I have checked and haven't found any problems or wrong colours or anything.

Reply:
Looks good, it seems dynamic resource is delayed